unison-src/transcripts/fix2567.output.md

```diff
@@ -27,10 +27,10 @@ structural ability Foo where
 
   ☝️  The namespace .somewhere is empty.
 
-.somewhere> view Foo
+.somewhere> view .some.subnamespace.Foo
```

Comment:
This error case this transcript is testing was concerned with name selection;
which we now test in multiple other transcripts and handle with biasing now anyways.